### PR TITLE
remove elog messages for app-editors/vscode

### PR DESCRIPTION
**Pull request by berinaniesh**
Closes: https://bugs.gentoo.org/904393

### CODE REVIEW

**Comment by thesamesam** [comment]
Please use `pkgdev commit` to both get the commit message format right and also add a Closes: tag for the bug you filed.

File: app-editors/vscode/vscode-1.77.1.ebuild [hunk 1]
```diff
@@ -122,7 +122,5 @@ src_install() {
 
 pkg_postinst() {
 	xdg_pkg_postinst
-	elog "You may want to install some additional utils, check in:"
```
Comment (thesamesam) [inline app-editors/vscode/vscode-1.77.1.ebuild:125]: You're sure you want to remove it entirely, rather than only showing it on first emerge?

Comment (berinaniesh) [reply app-editors/vscode/vscode-1.77.1.ebuild:125]: Its my personal opinion, but I think most people know how vscode works. I don't think it should be shown in the first emerge as well. It just wastes time.

**Comment by berinaniesh** [comment]
> Please use `pkgdev commit` to both get the commit message format right and also add a Closes: tag for the bug you filed.

The repository was quite large and so I used github codespaces to edit the files. Sorry about that and will use pkgdev next time. Is that fine @thesamesam

**Comment by thesamesam** [comment]
> > Please use `pkgdev commit` to both get the commit message format right and also add a Closes: tag for the bug you filed.
> 
> The repository was quite large and so I used github codespaces to edit the files. Sorry about that and will use pkgdev next time. Is that fine @thesamesam

The current PR doesn't have a signoff or correct commit message, unfortunately.

**Comment by berinaniesh** [comment]
I'll submit a new one.

**Pull request closed without merging**